ex03.exs

```diff
@@ -62,7 +62,19 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
-    « your code here »
```

Comment:
Why not use a pipeline for everything:

~~~
collection
|> Enum.chunk_every(...)
|> Enum.map(...)
etc
~~~

### -2